collect_app/src/main/res/drawable/settings_color.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
should it be in res/color/ folder instead of drawable? given that it is a colorr selector

Reply:
You are right.